code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -62,7 +62,7 @@ const styles = {
   },
   changeFocusArea: {
     fontFamily: '"Gotham 5r", sans-serif',
-    color: color.black,
+    color: color.dark_charcoal,
```

Comment:
I think what we really want is the [default text color `#333`](https://github.com/code-dot-org/code-dot-org/blob/b3794b7e6ab42375d2a922c7f4e02d70c783397a/dashboard/app/assets/stylesheets/scaffolds.css.scss#L3).